### ISSUE TITLE
Reconsider cadence?

### ISSUE THREAD

**Issue by s3ththompson** [s3ththompson/berliner#73]
An argument against passing in Berliner cadence as an option: `berliner.Daily` will likely be the most common usage.  But daily will throw away important articles from low-frequency sources if the user decides not to create/read a Berliner on the day the article is published.  This is not ideal.  Maybe the cadence should be fuzzy, and `sources.Dedupe` will naturally de-fuzz?

**Comment by geoffreylitt** [comment]
Yeah, this definitely gets at some of the limitations of just passing in a hardcoded date limit.

One possibility would be to split into two components:
- An extremely loose hard date limit -- e.g. "posted in the last few months" -- just to stop me from getting super old stuff.
- A filter that emphasizes recency. I would apply this to many sources but not necessarily to all (e.g., I only want NYT articles from the past couple days, but for certain blogs I'm okay seeing much older, less time-sensitive content)

This does raise performance concerns around fetching a ton of articles, but caching should help a bit.

Another possibility (which we've discussed) would be to re-architect sources by splitting out the fetching/scraping phases of sources as we had it before, and let scraping occur lazily. This would eliminate the need for a date limit, and would offer major performance improvements for cases like a source that's immediately clamped to only provide one Post. But this is more work, and probably not worth doing now. (in particular, not sure how important performance is -- if you're just running this every day on a server you don't really care)
